### PR TITLE
Spatial structure 1

### PR DESCRIPTION
**Pull request by seb-esser**
## Unit test

transporting an old example to the new repo. 

## Covered topics

*tick one/many*

- [x] project setup
- [ ] georeferencing
- [ ] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [x] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

**Review by larswik — APPROVED** [review]

**Review by larswik — COMMENTED** [review]
I wonder if the IfcBridge facility needs a composition type of ELEMENT since it is contained in an IfcRoad facility with composition type COMPLEX?

File: SpatialStructure-1/SpatialStructure1.ifc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#20= IFCAXIS2PLACEMENT3D(#19,$,$);
+#21= IFCLOCALPLACEMENT(#3,#20);
+#22= IFCFACILITY('2iR3QiEKT13xva7U_z3X$2',#10,'TrafficWayA',$,$,#21,$,$,.COMPLEX.);
+#23= IFCRELAGGREGATES('3gMREsTI573e1$2pf8yStl',#10,$,$,#4,(#22,#34));
+#24= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#25= IFCAXIS2PLACEMENT3D(#24,$,$);
+#26= IFCLOCALPLACEMENT(#21,#25);
+#27= IFCFACILITYPART('3358p6WkD51QxLDNAmC47A',#10,'myRoadPart01','TrafficWayA -> Segment 1',$,#26,$,$,$,IFCROADPARTTYPEENUM(.ROADSEGMENT.),.LONGITUDINAL.);
+#28= IFCRELAGGREGATES('3R1y780Mv2WOM1Ee_W8nTz',#10,$,$,#22,(#27,#29,#33));
+#29= IFCBRIDGE('2L4$ZHSpTDShVW6hQEDuP9',#10,'myBridge','TrafficWayA -> Segment 2',$,$,$,$,$,$);
```
Comment (larswik) [inline SpatialStructure-1/SpatialStructure1.ifc:45]: Composition type ELEMENT?

**Comment by pjanck** [comment]
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

File: SpatialStructure-1/SpatialStructure1.ifc (hunk 1 above)
Comment (AlexBrad1eyCT) [reply SpatialStructure-1/SpatialStructure1.ifc:45]: The default if omitted is ELEMENT

**Pull request merged**